koku/reporting/migrations/0038_auto_20190220_1511.py

```diff
@@ -0,0 +1,290 @@
+# Generated by Django 2.1.5 on 2019-02-20 15:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reporting', '0037_auto_20190218_2054'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            """
+            DROP VIEW IF EXISTS reporting_ocpawscostlineitem_daily;
+            DROP VIEW IF EXISTS reporting_ocpawsusagecostlineitem_daily;
+
+            CREATE OR REPLACE VIEW reporting_ocpawsusagelineitem_daily AS (
+                WITH cte_usage_tag_matched as (
+                    SELECT aws.id as aws_id,
+                            ocp.id as ocp_id,
+                            aws.usage_start,
+                            ocp.namespace
+                        FROM (
+                        SELECT aws.id,
+                            aws.usage_start,
+                            LOWER(key) as key,
+                            LOWER(value) as value
+                            FROM reporting_awscostentrylineitem_daily as aws,
+                                jsonb_each_text(aws.tags) labels
+                        ) AS aws
+                        JOIN (
+                            SELECT ocp.id,
+                                ocp.usage_start,
+                                ocp.cluster_alias,
+                                ocp.node,
+                                ocp.namespace,
+                                LOWER(key) as key,
+                                LOWER(value) as value
+                            FROM reporting_ocpusagelineitem_daily as ocp,
+                                jsonb_each_text(ocp.pod_labels) labels
+                        ) AS ocp
+                            ON aws.usage_start::date = ocp.usage_start::date
+                                AND (
+                                    (aws.key = ocp.key AND aws.value = ocp.value)
+                                    OR (aws.key = 'ocp_cluster' AND aws.value = ocp.cluster_alias)
```

Comment:
Sorry about this, but per Dayle's comments we should use `openshift` not `ocp` so let's update these to:
- `openshift_project`
- `openshift_node`
- `openshift_cluster`

Reply:
Good catch!